packages/flutter/lib/src/material/material.dart

```diff
@@ -180,6 +187,7 @@ class Material extends StatefulWidget {
     description.add(new EnumProperty<MaterialType>('type', type));
     description.add(new DoubleProperty('elevation', elevation));
```

Comment:
We're not sure why this doesn't have a default value, is that correct?

Reply:
Yeah, I think it should be `0.0`.

Reply:
dba7e0ced5eadda5bdf00a072ae2bf9f06517627